content/applications/hr/frontdesk.rst

```diff
@@ -0,0 +1,318 @@
+The information requested is what was configured for that specific frontdesk station. If any
+information is required, the field displays a red asterisk (*). The visitor must enter information
+in order to check in. Once all the information is entered, the visitor taps the :guilabel:`Check In`
+button.
+
+If drinks were configured for the station, after tapping :guilabel:`Check In`, a registration
+confirmation screen loads, along with the question `Do you want something to drink?`. The visitor
+can tap either :guilabel:`Yes, please`, or :guilabel:`No, thank you`. If they select :guilabel:`Yes,
+please`, a drink selection screen appears, and the pre-configured options are listed. The visitor
+then taps the desired selection, or if they do not want anything, they can tap the
+:guilabel:`Nothing, thanks` button at the bottom of the screen. If a drink selection was made, a
+`Thank you for registering! Your drink is on the way.` message appears.
+
+At any point in the check-in process, if ten (10) seconds pass with no selection, the kiosk returns
+to the main welcome screen.
+
+Once the visitor has checked in, both the person they are visiting, and any users who were
+configured to be notified when anyone checks-in at the kiosk are notified. The notification is
+either by email, SMS message, a *Discuss* chat, or any combination of those three options.
+
+If the visitor requested a drink, the user or users who were configured as the :guilabel:`People to
+Notify` on the drink form, are notified via the *Discuss* application. The message that appears is:
+`(Visitor Name) just checked-in. They requested (drink).`
+
+Once the drink has been delivered to the guest, the person who delivered the drink is responsible
+for marking the drink as delivered. To do this, navigate to :menuselection:`Frontdesk application
+--> Stations --> (X) Drinks to serve`. This opens a list of all the visitors who checked-in at that
+station, and are waiting for a drink. Click the :guilabel:`Drink Served` button at the end of the
+line for the visitor who was served. Once they are marked as having their drink served, the visitor
+disappears from the list.
+
+Visitors do **not** check themselves out when they leave. Once the visitor completed their business
+and has left the premises, it is important to check them out for accurate record keeping. Navigate
+to :menuselection:`Frontdesk application --> Stations --> (X) On Site`. This opens a list of all the
+visitors who are currently checked-in at that station. Click the :guilabel:`Check Out` button near
+the end of the line for the visitor who left. Once they are marked as checked-out, the visitor
+disappears from the list.
+
+.. important::
+   Always have an accurate list of who is on the premises at any given time. Since visitors do not
+   check themselves out, it is important for accurate record keeping to check-out visitors when they
+   leave. This is important both in the case of an emergency, and for security purposes.
```

Comment:
```suggestion
   Always have an accurate list of who is on the premises at any given time. Since visitors do not
   check themselves out, it is important to check out visitors when they leave, for accurate record keeping. This is 
   important both in the case of an emergency, and for security purposes.
```